packages/ui/src/components/vuestic-components/va-switch/VaSwitch.vue

```diff
@@ -287,20 +304,57 @@ export default {
     box-shadow: inset 0 2px 4px rgba(0, 0, 0, 0.1);
     transition: background-color 0.2s ease;
 
-    &--error {
-      border: 2px solid $theme-red;
+    @at-root {
+      .va-switch--on-keyboard-focus#{&} {
+        transition: all, 0.6s, ease-in;
+        box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.3);
+      }
+      .va-switch--small#{&} {
+        border-radius: 0.75rem;
+      }
+      .va-switch--large#{&} {
+        border-radius: 1.25rem;
+      }
     }
   }
 
-  &__track-label {
+  #{&}__track-label {
     color: $white;
+    margin: auto 7px auto 33px;
```

Comment:
can't we do rems?
Also in other places?
Ideally we want rems except for cases, where we can't do rems.